### DISCUSSION TITLE
Platforms Starter Kit New Site Not Resolving

### DISCUSSION THREAD

**Discussion by maxforbang** [vercel/community#6237] [Help] [answered]
### Summary

My deployment for the [Platforms Starter Kit](https://vercel.com/templates/next.js/platforms-starter-kit) works fine on Vercel.

However, when I create a new site through the UI, going to that new site (test.mydomain.com) doesn't resolve. Doesn't even give a 404.

Do I also need to add this domain's A to my DNS records?

### Steps to Reproduce

1. Deploy the Platforms Starter Kit
2. Create a new site through the UI
3. Navigate to that site (test.rootdomain.com)

**Comment by versecafe** [comment, marked as answer]
@maxforbang You need to make sure you reserver `*.yourdomain.com` to allow for the dynamic * assignments
